angularjs-portal-home/src/main/webapp/js/override.js

```diff
@@ -28,11 +25,17 @@ define(['angular'], function(angular) {
                 'entry': 'entry/',
                 'entries': 'entries.json',
             },
-            'notificationsURL': '/web/staticFeeds/notifications.json',
+            'messagesURL': '/web/staticFeeds/messages.json',
             'groupURL': '/portal/api/groups',
             'kvURL': '/storage',
             'loginSilentURL': '/portal/Login?silent=true&profile=bucky',
           },
+          'MESSAGES': {
+            'announcementsEnabled': true,
+            'notificationsEnabled': true,
+            'groupFiltering': true,
```

Comment:
Do we need this flag, or could group filtering be implied by whether there are groups in the items in the messages feed? If groups are specified, filter to them. If they're not, don't worry about it.